eldritchmush/commands/combat_commands/battlefield_medicine.py

```diff
@@ -7,8 +7,10 @@ class CmdBattlefieldMedicine(Command):
     key = "medic"
     help_category = "mush"
 
+    def __init__(self):
```

Comment:
Don't need to do this with Evennia. Look at the out of the box command class documentation for a lot of cool built in stuff they did, the parse class method being one of them.

Reply:
I commented on this later. I like that you're initializing the variables though now it will be harder to debug my code hah!